### PR TITLE
[com_banners]Code style

### PR DESCRIPTION
**Pull request by wojsmol**
### Summary of Changes

Code style changes for com_banners
### Testing Instructions

code review

cc @andrepereiradasilva @JoomliC @brianteeman

### CODE REVIEW

File: administrator/components/com_banners/models/forms/banner.xml [hunk 1]
```diff
@@ -1,224 +1,376 @@
...
+			size="20"
+		/>
 
-		<field name="modified" type="calendar" class="readonly"
-			label="JGLOBAL_FIELD_MODIFIED_LABEL" description="COM_BANNERS_FIELD_MODIFIED_DESC"
-			size="22" readonly="true" format="%Y-%m-%d %H:%M:%S" filter="user_utc" />
+		<field
+			name="modified"
+			type="calendar"
+			class="readonly"
```
Comment (andrepereiradasilva) [inline administrator/components/com_banners/models/forms/banner.xml]: move class down?

File: administrator/components/com_banners/models/forms/banner.xml [hunk 2]
```diff
@@ -1,224 +1,376 @@
...
+			type="impmade"
+			label="COM_BANNERS_FIELD_IMPMADE_LABEL"
+			description="COM_BANNERS_FIELD_IMPMADE_DESC"
+			default="0"
+		/>
+
+		<field
+			name="clicks"
+			type="clicks"
+			default="0"
```
Comment (andrepereiradasilva) [inline administrator/components/com_banners/models/forms/banner.xml]: move default down?

File: administrator/components/com_banners/models/forms/banner.xml [hunk 3]
```diff
@@ -1,224 +1,376 @@
...
-				class="input-mini validate-numeric" label="COM_BANNERS_FIELD_WIDTH_LABEL"
-				description="COM_BANNERS_FIELD_WIDTH_DESC" />
-
-			<field name="height" type="text"
-				class="input-mini validate-numeric" label="COM_BANNERS_FIELD_HEIGHT_LABEL"
-				description="COM_BANNERS_FIELD_HEIGHT_DESC" />
-
-			<field name="alt" type="text"
-				label="COM_BANNERS_FIELD_ALT_LABEL" description="COM_BANNERS_FIELD_ALT_DESC" />
+				description="COM_BANNERS_FIELD_IMAGE_DESC"
```
Comment (andrepereiradasilva) [inline administrator/components/com_banners/models/forms/banner.xml]: move label and decription after type?

File: administrator/components/com_banners/models/forms/client.xml [hunk 4]
```diff
@@ -41,52 +65,58 @@
...
+			<option value="4">COM_BANNERS_FIELD_VALUE_WEEKLY</option>
+			<option value="5">COM_BANNERS_FIELD_VALUE_DAILY</option>
 		</field>
 
-		<field name="track_impressions" type="list" default="0"
-			class="chzn-color"
+		<field
+			name="track_impressions"
+			type="list"
+			default="0"
```
Comment (andrepereiradasilva) [inline administrator/components/com_banners/models/forms/client.xml]: move default down?

File: administrator/components/com_banners/models/forms/client.xml [hunk 5]
```diff
@@ -41,52 +65,58 @@
...
-		<field name="metakey" type="textarea"
-			rows="3" cols="30" label="JFIELD_META_KEYWORDS_LABEL"
-			description="COM_BANNERS_FIELD_CLIENT_METAKEYWORDS_DESC" />
+	<fieldset name="metadata" label="JGLOBAL_FIELDSET_METADATA_OPTIONS">
+		<field
+			name="metakey"
+			type="textarea"
+			rows="3"
+			cols="30"
+			label="JFIELD_META_KEYWORDS_LABEL"
```
Comment (andrepereiradasilva) [inline administrator/components/com_banners/models/forms/client.xml]: rows and cols down?

File: administrator/components/com_banners/models/forms/client.xml [hunk 6]
```diff
@@ -41,52 +65,58 @@
...
+			description="COM_BANNERS_FIELD_CLIENT_METAKEYWORDS_DESC"
+		/>
 
 		<field name="own_prefix" type="radio"
 			class="btn-group btn-group-yesno"
```
Comment (andrepereiradasilva) [inline administrator/components/com_banners/models/forms/client.xml]: missed this one

File: administrator/components/com_banners/models/forms/client.xml [hunk 7]
```diff
@@ -98,18 +128,23 @@
...
+			name="extrainfo"
+			type="textarea"
 			class="span12"
-			rows="5" cols="80" label="COM_BANNERS_FIELD_EXTRAINFO_LABEL"
-			description="COM_BANNERS_FIELD_EXTRAINFO_DESC" />
-
+			rows="5"
+			cols="80"
+			label="COM_BANNERS_FIELD_EXTRAINFO_LABEL"
+			description="COM_BANNERS_FIELD_EXTRAINFO_DESC"
```
Comment (andrepereiradasilva) [inline administrator/components/com_banners/models/forms/client.xml]: class, rows and cols down?

**Comment by JoomliC** [comment]
@wojsmol I'm sorry, but i don't understand your workflow...

You apply code style here for some files (php + xml) but you do some and not all...
Why?

Isn't it easier to focus on all php or xml files inside on component, instead of doing some but not all, and mixing php and xml... This does not help to know if one extension is really full reviewed or not...
I would recommend to focus first on all xml (as i did for (first in admin) com_categories, com_newsfeeds, com_contact). Of course only my point of view, but IMO, could be good to know what's done in a PR, and what's to be done...

;-)

**Comment by brianteeman** [comment]
That's my fault. I asked for all the remaining fixes to be on one pr after
discussing with some of the committers during jab.

**Comment by JoomliC** [comment]
Thanks @brianteeman !
So what is the rule there, now ? About code style ?

@wojsmol see you just added other not reviewed files 👍 
Ping me when done, will review ;-)

**Comment by brianteeman** [comment]
The idea is that once you have done a few small ones to establish the style
/ changes and they have been accepted then the rest can be in a single pr.
This will hopefully reduce everyone's workload. If it doesn't work we can
always review the process.

**Comment by JoomliC** [comment]
@brianteeman I agree! 👍 
This why i asked about not reviewed files in the com_banners, and only some reviewed... (seems to me as not a ready to be reviewed PR...).

So, in case of components and code styles, could be conmplex to do all components in the same PR.
- Should we split each component code style PR ? (seems more easy to prevent conflict, and help review).
- Should we split php and xml files code style review ?

About this current PR, my issue was mainly to see not all xml files and not all php files reviewed ;-)

**Comment by wojsmol** [comment]
@JoomliC Now I corrected all the com_banners files

**Comment by JoomliC** [comment]
@wojsmol just started to check config.xml, and see at start a not reviews field... Could you check your changes before? Thanks!

File: administrator/components/com_banners/config.xml [hunk 8]
```diff
@@ -1,39 +1,36 @@
...
-			<option
-				value="1">JYES</option>
-			<option
-				value="0">JNO</option>
+			>
+			<option value="1">JYES</option>
+			<option value="0">JNO</option>
 		</field>
 
 		<field
```
Comment (JoomliC) [inline administrator/components/com_banners/config.xml]: This field is not reviewed and should...

Comment (JoomliC) [inline administrator/components/com_banners/config.xml]: One field forgotten here...

File: administrator/components/com_banners/models/forms/banner.xml [hunk 9]
```diff
@@ -1,224 +1,376 @@
...
 	<fields name="params" label="JGLOBAL_FIELDSET_DISPLAY_OPTIONS">
 		<fieldset name="image">
-			<field name="imageurl" type="media" directory="banners"
-				hide_none="1" label="COM_BANNERS_FIELD_IMAGE_LABEL"
+			<field
+				name="imageurl"
+				type="media"
+				directory="banners"
+				hide_none="1"
+				label="COM_BANNERS_FIELD_IMAGE_LABEL"
```
Comment (JoomliC) [inline administrator/components/com_banners/models/forms/banner.xml]: Move label up...

File: administrator/components/com_banners/models/forms/filter_banners.xml [hunk 10]
```diff
@@ -27,6 +29,7 @@
 			>
 			<option value="">JOPTION_SELECT_CATEGORY</option>
 		</field>
+
 		<field
          	name="client_id"
```
Comment (JoomliC) [inline administrator/components/com_banners/models/forms/filter_banners.xml]: indent...

**Comment by wojsmol** [comment]
@JoomliC  I'll check as soon as github starts to function normally https://status.github.com/messages<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10602">issues.joomla.org/joomla-cms/10602</a>.</sub>

**Comment by JoomliC** [comment]
@wojsmol Yes, not working well today ;-) (for me, seems just back ok after 1 hour with no synchronization possible...)

**Comment by wojsmol** [comment]
@JoomliC I reviewed all the files again

File: administrator/components/com_banners/banners.xml [hunk 11]
```diff
@@ -35,14 +35,37 @@
...
+				alt="Banners/Clients"
+				>
+				com_banners_clients
+			</menu>
+			<menu
+				link="option=com_banners&amp;view=tracks"
+				view="tracks"
+				img="class:banners-tracks"
+				alt="Banners/Tracks"
+				>com_banners_tracks
```
Comment (JoomliC) [inline administrator/components/com_banners/banners.xml]: The com_banners_tracks is not on a new line as for others.

**Comment by JoomliC** [comment]
@wojsmol Just added one note on banners.xml
Everything else seems ok! 👍

**Comment by wojsmol** [comment]
@JoomliC New line added :smile:

**Comment by JoomliC** [comment]
I have tested this item :white_check_mark: successfully on 02064d6d29ffa65c7913e479eafcc2a873eb2d17<br /><br />On code review

@wojsmol Thanks!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10602">issues.joomla.org/joomla-cms/10602</a>.</sub>

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 02064d6d29ffa65c7913e479eafcc2a873eb2d17<br /><br />on code review
thanks<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10602">issues.joomla.org/joomla-cms/10602</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10602">issues.joomla.org/joomla-cms/10602</a>.</sub>

**Pull request closed without merging**